flare/algorithm_zoo/simple_algorithms.py

```diff
@@ -225,7 +242,7 @@ def learn(self, inputs, next_inputs, states, next_states, next_alive,
                                            self.grad_clip)
         self.optim.step()
 
-        return dict(cost=cost), states_update, next_states_update
+        return dict(cost=cost)
```

Comment:
You might want to add the condition of states for all the algorithms implemented.

Reply:
I can use a separate pr to do that